### PR TITLE
Features/24 testimonial page and faq

### PR DESCRIPTION
**Pull request by BaptisteLegat**

### CODE REVIEW

File: app/(base-layout)/faq/page.tsx
Comment (Dercraker) [inline app/(base-layout)/faq/page.tsx:1]: niveau desing sort la faq du paper

File: app/(base-layout)/feedback/_component/FeedbackCarousel.tsx [hunk 1]
```diff
@@ -0,0 +1,83 @@
...
+
+  const {data: feedbacks = [], isPending} = useQuery({
+    queryKey: ['randomFeedback'],
+    queryFn: async () => {
+      const { data: feedbacks, serverError } = await GetRandomFeedBackAction({
+        take: 50,
+      });
+
+      if (!feedbacks || serverError) {
+        //TODO: Error notify
```
Comment (Dercraker) [inline app/(base-layout)/feedback/_component/FeedbackCarousel.tsx:30]: make this

File: app/(base-layout)/feedback/_component/FeedbackCarousel.tsx [hunk 2]
```diff
@@ -0,0 +1,83 @@
...
+import Autoplay from 'embla-carousel-autoplay';
+import { useRef } from 'react';
+import type { RandomFeedback } from '@/features/feedback/get/getRamdomFeedback.query';
+import { GetRandomFeedBackAction } from '@/features/feedback/get/getRandomFeedback.action';
+import { useQuery } from '@tanstack/react-query';
+
+export const FeedbackCarousel = () => {
+  const autoplay = useRef(Autoplay({ delay: 5000 }));
+
+  const {data: feedbacks = [], isPending} = useQuery({
```
Comment (Dercraker) [inline app/(base-layout)/feedback/_component/FeedbackCarousel.tsx]: Add error

File: src/components/feedback/ContactFeedbackMenu.tsx
Comment (Dercraker) [inline src/components/feedback/ContactFeedbackMenu.tsx:1]: camelcase

File: app/(base-layout)/feedback/_component/FeedbackCarousel.tsx
Comment (Dercraker) [inline app/(base-layout)/feedback/_component/FeedbackCarousel.tsx:1]: camelcase

File: app/(dashboard-layout)/_component/layout/DashboardHeader.tsx
Comment (Dercraker) [inline app/(dashboard-layout)/_component/layout/DashboardHeader.tsx:1]: camelcase

File: src/components/feedback/FeedbackReviewInput.tsx
Comment (Dercraker) [inline src/components/feedback/FeedbackReviewInput.tsx:1]: check si c'est possible d'utiliser lordicon

**Review by Dercraker — APPROVED** [review]

File: src/components/feedback/FeedbackReviewInput.tsx
Comment (BaptisteLegat) [reply src/components/feedback/FeedbackReviewInput.tsx:1]: not possible now

**Pull request merged**